From 8857767d5b2f5a035321e3712e8cd0008840ba6b Mon Sep 17 00:00:00 2001
From: Andrii Vasylevskyi <avasylev@gmail.com>
Date: Mon, 9 Dec 2019 20:50:09 -0500
Subject: [PATCH 14/24] Add wasm32 as supported platform (#120)

Summary:
Emscripten is already included, adding wasm32 the same way for when
build with "plain" clang wasm32 (without emscripten)

Test Plan:
make
test/cctest/cctest --list | tr -d '<' | xargs test/cctest/cctest

Reviewers:

Subscribers:

Tasks:

Tags:
---
 double-conversion/utils.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/double-conversion/utils.h b/double-conversion/utils.h
index 98ec28b..5393baa 100644
--- a/double-conversion/utils.h
+++ b/double-conversion/utils.h
@@ -104,7 +104,7 @@ int main(int argc, char** argv) {
     defined(__riscv) || defined(__e2k__) || \
     defined(__or1k__) || defined(__arc__) || \
     defined(__microblaze__) || defined(__XTENSA__) || \
-    defined(__EMSCRIPTEN__)
+    defined(__EMSCRIPTEN__) || defined(__wasm32__)
 #define DOUBLE_CONVERSION_CORRECT_DOUBLE_OPERATIONS 1
 #elif defined(__mc68000__) || \
     defined(__pnacl__) || defined(__native_client__)
-- 
2.29.2

